qemu/tests/cfg/x86_cpu_flags.cfg

```diff
@@ -84,6 +84,10 @@
                             cpu_model_flags += ",+pcid,+invpcid"
         - others:
             variants:
+                - test_default_model:
+                    auo_cpu_model = no
+                    origin_model = yes
```

Comment:
If default model is fixed as qemu64, why not set `origin_model = qemu64` directly in cfg?

Reply:
> If default model is fixed as qemu64, why not set `cpu_model = qemu64` directly in cfg?

In case it changes later.

Reply:
`cpu_model = ${default_cpu_model}`?

Reply:
> $

Seems this doesn't work if we don't define the "default_cpu_model" in the same cfg file, it will use the latest cpu model.

Reply:
> > $
> 
> Seems this doesn't work if we don't define the "default_cpu_model" in the same cfg file, it will use the latest cpu model.

@vivianQizhu Would you please give some advice? Thanks.

Reply:
@nanliu-r Sorry for my late response, if so, I suppose `auto_cpu_model = no`  + `cpu_model = ""` will make it work as you expected. By the way there is a typo `auo`.

Reply:
> @nanliu-r Sorry for my late response, if so, I suppose `auto_cpu_model = no` + `cpu_model = ""` will make it work as you expected. By the way there is a typo `auo`.

@vivianQizhu sorry, update and test PASS, thanks!

 (1/1) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.x86_cpu_flags.others.test_default_model: PASS (92.29 s)